e2e/tests/wasm/grandpa_test.go

```diff
@@ -383,25 +310,26 @@ func (s *GrandpaTestSuite) fundUsers(t *testing.T, ctx context.Context, fundAmou
 	return polkadotUser, cosmosUser
 }
 
-func modifyGenesisShortProposals(votingPeriod string, maxDepositPeriod string) func(ibc.ChainConfig, []byte) ([]byte, error) {
-	return func(chainConfig ibc.ChainConfig, genbz []byte) ([]byte, error) {
-		g := make(map[string]interface{})
-		if err := json.Unmarshal(genbz, &g); err != nil {
-			return nil, fmt.Errorf("failed to unmarshal genesis file: %w", err)
-		}
-		if err := dyno.Set(g, votingPeriod, "app_state", "gov", "params", "voting_period"); err != nil {
-			return nil, fmt.Errorf("failed to set voting period in genesis json: %w", err)
-		}
-		if err := dyno.Set(g, maxDepositPeriod, "app_state", "gov", "params", "max_deposit_period"); err != nil {
-			return nil, fmt.Errorf("failed to set max deposit period in genesis json: %w", err)
-		}
-		if err := dyno.Set(g, chainConfig.Denom, "app_state", "gov", "params", "min_deposit", 0, "denom"); err != nil {
-			return nil, fmt.Errorf("failed to set min deposit in genesis json: %w", err)
-		}
-		out, err := json.Marshal(g)
-		if err != nil {
-			return nil, fmt.Errorf("failed to marshal genesis bytes to json: %w", err)
-		}
-		return out, nil
+// validateTestConfig ensures that the given test config is valid for this test suite.
+func validateTestConfig() {
+	tc := testsuite.LoadConfig()
+	if tc.ActiveRelayer != "hyperspace" {
+		panic(fmt.Errorf("hyperspace relayer must be specified"))
 	}
 }
+
+func getConfigOverrides() map[string]any {
+	consensusOverrides := make(testutil.Toml)
+	blockTime := 5 // seconds, parachain is 12 second blocks, don't make relayer work harder than needed
```

Comment:
I don't really understand why block time is 5 and the comment says 12

Reply:
ah, this bit actually just came from the original test, let me try and see if I can clean up the comments.